src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("사이즈는 3이다")
+    void size() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
```

Comment:
```suggestion
    void size() {
                assertThat(numbers).hasSize(3);
    }
```
알고 계실 수 있지만 assertj 에서는 Collection 에 대한 검증 기능도 지원되고 있습니다!!
중요한 부분은 아니니 참고만해주세요 😆

Reply:
잘 모르고 있던 내용인데 assertj 에서의 Collection 검증 기능 한번 보겠습니다! 감사합니다! 😄